batchflow/batch.py

```diff
@@ -634,7 +634,7 @@ def apply_parallel(self, func, init=None, post=None, src=None, dst=None, *args,
         # Compute result. Unbind the method to pass self explicitly
         parallel = inbatch_parallel(init=init, post=post, target=target, src=src, dst=dst)
         transform = parallel(type(self)._apply_once)
-        result = transform(self, *args, func=func, p=p, src=src, dst=dst,
+        result = transform(self, *args, func=func, p=p,
```

Comment:
Nope
#656 fixes the same problem by adding `src/dst` to the signatures of methods.

In (hopefully, not too distant) future I will rework the decorator to be a class that can use `inspect` to look at the method signature, see #672

Reply:
Well, in that case all `ImagesBatch` methods need `src` and `dst` arguments. I guess it's a 2-minute fix. But why do you even need those parameters inside batch actions anyway?

Reply:
This way, we can use `apply_parallel` decorator everywhere, instead of using `inbatch_parallel` somewhere (like it was previously)